docs/designers-developers/developers/themes/block-based-themes.md

```diff
@@ -0,0 +1,92 @@
+# Block-based themes (Experimental)
+
+> This is the documentation for the current implementation of the block-based themes, also known as Full Site Editing or Block Content Areas. These features are still experimental in the plugin. “Experimental” means this is just an early implementation that is subject to potential drastic and breaking changes in iterations based on feedback from users, contributors and theme authors.
+
+> Documentation has been shared early to surface what’s being worked on and invite feedback from those experimenting with the APIs. You can provide feedback in the weekly #core-editor chats where the latest progress of this effort will be shared and discussed, or async via Github issues.
+
+## What is a block-based theme?
+
+A block-based theme is a WordPress theme that is entirely composed of blocks. In addition to the post content of the different Post Types (pages, posts, ...), the block editor can also be used to edit all areas of the site: headers, footers, sidebars,...
+
+## What is the structure of a block-based theme?
+
+A very simple block-based theme is structured like so:
+
+```
+theme
+|__ style.css
+|__ functions.php
+|__ block-templates
+    |__ index.html
+    |__ single.html
+    |__ archive.html
+    |__ ...
+|__ block-template-parts
+    |__ header.html
+    |__ footer.html
+    |__ sidebar.html
+    |__ ...
+```
+
+The difference with existing WordPress themes is that the different templates in the template hierarchy are block templates instead of php files.
```

Comment:
```suggestion
The difference with existing WordPress themes is that the different templates in the template hierarchy, and template parts, are block templates instead of php files.
```